Kconfig.nrf

```diff
@@ -16,14 +33,19 @@ config BOARD_THINGY91_NRF9160NS
 #  - For CC3XX RNG
 config MAIN_STACK_SIZE
 	default 2048 if ZTEST
-	default 2048 if ENTROPY_CC3XX
+	default 2048 if ENTROPY_CC3XX && !(SPM || BUILD_WITH_TFM)
```

Comment:
Nit: This could use a comment about why SPM and TFM are excluded.

Reply:
Will be fixed after this is merged